spec/models/manageiq/providers/autosde/inventory_spec.rb

```diff
@@ -23,5 +23,7 @@
     expect(ems.cloud_volumes).to_not be_empty
 
     expect(ems.cloud_volumes.first).to be_instance_of(ManageIQ::Providers::Autosde::StorageManager::CloudVolume)
+    expect(ems.storage_hosts_wwpn_candidates).to_not be_empty
```

Comment:
Ideally we should be checking the attributes for one of the records that gets created.  This helps ensure that e.g. the lazy_find association was built correctly